string session_id
string peer_id
string label
int32 id
bool reliable
bool ordered
string protocol
---
